package.json

```diff
@@ -174,7 +176,8 @@
     "typedoc": "^0.12.0",
     "typescript": "3.4.5",
     "uglify-es": "3.3.9",
-    "uglifyjs-webpack-plugin": "2.1.2"
+    "uglifyjs-webpack-plugin": "2.1.2",
```

Comment:
Why also add `uglifyjs-webpack-plugin`?

Reply:
we didn't, the , was added, that's why there is a change here

Reply:
sorry 🙂